src/js/containers/award/shared/ContractGrantActivityContainer.jsx

```diff
@@ -187,6 +187,7 @@ const ContractGrantActivityContainer = ({ awardId, awardType, dates }) => {
             !error.error
             && !loading
             && transactions.length > 0
+            && !badDates
```

Comment:
What if the potential end and current end dates are the same?

Reply:
Yeah, that is an edge case we have in Dev-4553